### DISCUSSION TITLE
Set cron string as output from cat/echo

### DISCUSSION THREAD

**Discussion by victor-mariano-leite** [community/community#27188] [Actions]
<p>Good morning/evening/night everyone! :slight_smile:</p>
<p>I’ve been trying to pass a cron string to an output with:</p>
<pre><code class="lang-auto">name: Test
on:
  push:
    branches: ["main"]
jobs:
  job1:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - id: step1
        name: step1 test
        env:
          PATH: ./cron
        run: |
          echo "::set-output name::cron=$(cat $PATH)"
      - id: step2
        name: step2 test
        run: |
          echo ${{ steps.step1.outputs.cron }}
</code></pre>
<p>And the cron file has only the schedule <code>20 * * * 4 *</code> for example.</p>
<p>Executing this, gives me this error:</p>
<pre><code class="lang-auto">Error: Unable to process command '::set-output name::cron=20 * * * 4 *' successfully.

[7](https://github.com/&lt;USER&gt;/&lt;REPOSITORY&gt;/runs/1560583527?check_suite_focus=true#step:4:7)Error: Required field 'name' is missing in ##[set-output] command.
</code></pre>
<p>Anyone knows the reason of this behavior?</p>

**Comment by jvalkeal** [comment, marked as answer]
<p>Maybe you already figured it out but syntax is wrong as it’s:</p>
<p><code>::set-output name={name}::{value}</code></p>
